home/admin.py

```diff
@@ -6,3 +6,8 @@
 @admin.register(PageView)
 class PageViewAdmin(admin.ModelAdmin):
     list_display = [field.name for field in PageView._meta.get_fields()]
+
+
+# @admin.register(WhatsAppTemplate)
+# class WhatsappTemplateAdmin(admin.ModelAdmin):
+#     list_display = [field.name for field in WhatsAppTemplate._meta.get_fields()]
```

Comment:
Do we need this? If not, let's delete it. (If we're not sure yet, maybe add a TODO to clean it up so we don't forget.)

Reply:
Not needed, removed